backend/cmd/main.go

```diff
@@ -20,11 +20,24 @@ func main() {
 func run() error {
 	// GitHub GraphQL API v4 token.
 	c := helpers.NewGHGraphQLClient()
-	githubCollector := github.NewGitHubCollector(c)
+	org := os.Getenv("ORGANIZATION_NAME")
+	if org == "" {
+		org = "sbv-world-health-org-metrics"
+	}
+	githubCollector := github.NewGitHubCollector(c, org)
 	core := core.New(githubCollector)
 
-	results, errors := core.Amass(context.Background())
-	fmt.Println(results)
-	fmt.Println(errors)
+	errors := core.Amass(context.Background())
+
+	// We should really retry on error, and preserve the data we've already collected
```

Comment:
By preserve the data we've already collected, I'm guessing that means any of the fetchers that were successfully able to query and build the JSON during this run? If that's the case, we will have to have some logic to merge the existing data with this new data rather than overwrite the file that way we would have stale data rather than missing data for the fetcher(s) that failed.

Reply:
But I agree that we should have some sort of retry mechanism at the very least.

Reply:
Yeah, I think we probably need to define what `"the data we've already collected"` means. This might be more helpful in a future state where we have multiple collectors - a failure in one collector shouldn't stop us from updating datasets for other collectors. 

I think I'll delete this comment and instead replace it with an issue so we can more easily keep track of this as a potential TODO.

Reply:
#66 for reference!